malariagen_data/pf7.py

```diff
@@ -0,0 +1,314 @@
+            "FILTER_Centromere": [DIM_VARIANT],
+            "FILTER_InternalHypervariable": [DIM_VARIANT],
+            "FILTER_LowQual": [DIM_VARIANT],
+            "FILTER_Low_VQSLOD": [DIM_VARIANT],
+            "FILTER_MissingVQSLOD": [DIM_VARIANT],
+            "FILTER_Mitochondrion": [DIM_VARIANT],
+            "FILTER_SubtelomericHypervariable": [DIM_VARIANT],
+            "FILTER_SubtelomericRepeat": [DIM_VARIANT],
+            "FILTER_VQSRTrancheINDEL99.50to99.60": [DIM_VARIANT],
+            "FILTER_VQSRTrancheINDEL99.60to99.80": [DIM_VARIANT],
+            "FILTER_VQSRTrancheINDEL99.80to99.90": [DIM_VARIANT],
+            "FILTER_VQSRTrancheINDEL99.90to99.95": [DIM_VARIANT],
+            "FILTER_VQSRTrancheINDEL99.95to100.00+": [DIM_VARIANT],
+            "FILTER_VQSRTrancheINDEL99.95to100.00": [DIM_VARIANT],
+            "FILTER_VQSRTrancheSNP99.50to99.60": [DIM_VARIANT],
+            "FILTER_VQSRTrancheSNP99.60to99.80": [DIM_VARIANT],
+            "FILTER_VQSRTrancheSNP99.80to99.90": [DIM_VARIANT],
+            "FILTER_VQSRTrancheSNP99.90to99.95": [DIM_VARIANT],
+            "FILTER_VQSRTrancheSNP99.95to100.00+": [DIM_VARIANT],
+            "FILTER_VQSRTrancheSNP99.95to100.00": [DIM_VARIANT],
+            "FS": [DIM_VARIANT],
+            "ID": [DIM_VARIANT],
+            "InbreedingCoeff": [DIM_VARIANT],
+            "LOF": [DIM_VARIANT],
+            "MLEAC": [DIM_VARIANT, DIM_ALT_ALLELE],
+            "MLEAF": [DIM_VARIANT, DIM_ALT_ALLELE],
+            "MQ": [DIM_VARIANT],
+            "MQRankSum": [DIM_VARIANT],
+            "NEGATIVE_TRAIN_SITE": [DIM_VARIANT],
+            "NMD": [DIM_VARIANT],
+            "POSITIVE_TRAIN_SITE": [DIM_VARIANT],
+            "QD": [DIM_VARIANT],
+            "QUAL": [DIM_VARIANT],
+            "RAW_MQandDP": [DIM_VARIANT, DIM_PLOIDY],
+            "ReadPosRankSum": [DIM_VARIANT],
+            "RegionType": [DIM_VARIANT],
+            "SOR": [DIM_VARIANT],
+            "VQSLOD": [DIM_VARIANT],
+            "altlen": [DIM_VARIANT, DIM_ALT_ALLELE],
+            "culprit": [DIM_VARIANT],
+            "set": [DIM_VARIANT],
+        }
+
+    def _load_config(self, data_config):
+        """Load the config for data structure on the cloud into json format."""
+        if not data_config:
+            working_dir = os.path.dirname(os.path.abspath(__file__))
+            data_config = os.path.join(working_dir, "pf7_config.json")
+        with open(data_config) as pf7_json_conf:
+            config_content = json.load(pf7_json_conf)
+        return config_content
+
+    def sample_metadata(self):
+        """Access sample metadata and return as pandas dataframe
+
+        Returns
+        -------
+            df (pandas.DataFrame) : A dataframe of sample metadata on the samples that were sequenced as part of this resource.
+                                    Includes the time and place of collection, quality metrics, and accesion numbers.
+                                    One row per sample.
```

Comment:
Please edit to match [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) style.

Reply:
Should now all be in numpydoc style